doc/locale/ja/LC_MESSAGES/install/cmake_options.po

```diff
@@ -0,0 +1,161 @@
+# -*- po -*-
+# Japanese translations for Groonga package.
+# Copyright (C) 2009-2022 Groonga Project
+# This file is distributed under the same license as the Groonga package.
+# Automatically generated, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Groonga 12.0.8\n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: 2022-09-02 03:00+0900\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: ja\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+msgid "CMake options"
+msgstr "CMake オプション"
+
+msgid "This section describes important CMake options."
```

Comment:
```suggestion
msgid "This section describes important options for CMake."
```
With the current order of word, it is not clear that "important" modify. 

And If this options will be set in Groonga to use CMake, it would be nice to use "for using CMake" instead of "for CMake". If this options will be set in CMake within Groonga, no need to worry about. 

I hope you got  what I meant.

Reply:
These options will be set in CMake to make Groonga, so I adopt this suggestion.